2020-03-Autumn/1_pointer_techniques/QuestionOne/driver.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#import <vector>
+using namespace std;
+
+int removeDuplicates(std::vector<int>& nums) {
+
+    if(nums.size() == 0){
+        return 0;
+    }
+
+    //Pointer to the last, unique element in the array
+    int left = 0;
+    int right = 1;
+    int size = nums.size();
+
+    //Iterate through thr entire vector
+    while ( right < size ){
+
+        if ( nums[right] == nums[left]) {
+            right++;
+        }else{
+            left++;
+            nums[left] = nums[right];
+            right++;
+        }
+    }
+    //return the total unique elements in the array
+    return left + 1;
+}
+
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
+
+    vector<int> nums;
+    nums.push_back(1);
+    nums.push_back(1);
+    nums.push_back(2);
+
+    int unique = removeDuplicates(nums);
+
+    for (int i = 0; i < unique ; ++i) {
+        cout << nums[i] <<" ";
```

Comment:
Maybe add a comment here showing what the output should be do it's easy for someone to verify that they got the answer they wanted.